pkg/helm/list_releases.go

```diff
@@ -2,16 +2,20 @@ package helm
 
 import (
 	"helm.sh/helm/v3/pkg/action"
+	"helm.sh/helm/v3/pkg/release"
 )
 
 func ListReleases(conf *action.Configuration) (interface{}, error) {
 	cmd := action.NewList(conf)
-	//cmd.AllNamespaces = true
 	cmd.All = true
 
 	releases, err := cmd.Run()
 	if err != nil {
 		return nil, err
 	}
+	if releases == nil {
```

Comment:
the contract should be documented above - in case of no results, an empty list will be returned.

Reply:
done